src/main/java/org/sg/domain/Topic.java

```diff
@@ -0,0 +1,38 @@
+package org.sg.domain;
+
+public class Topic {
+	private int id;
+	private String name;
+	private String description;
+	private int course;
```

Comment:
perchè questa referenza al corso? un Topic può essere associato a piu di un corso. In questa logica, ci starebbe piu una lista dei corsi (e non un solo corso)....ma penso questa informazione sia più utile nell' oggetto Corso. Quindi meglio avere una lista di Topics nel Corso (e non il viceversa)

Reply:
ho tolto, in realtà ho preso spunto dal xml-campus che si era fatto, e si era associata una variabile course al topic